src/instructlab/eval/mmlu.py

```diff
@@ -153,7 +153,9 @@ def run(self, server_url: str | None = None) -> tuple:
 
         return overall_score, individual_scores
 
-    def _run_mmlu(self, server_url: str | None = None) -> dict:
+    def _run_mmlu(
```

Comment:
Rather than change the return value here, would you mind making this method return mmlu_output and create a wrapper method called by this class to pull "results" out of it.

Reply:
if it's ok, I think a wrapper method is not needed, instead I added the following in the mmlu run() method:
for task, result in results **['results']** .items():